frontend/src/components/Gallery/_Gallery.scss

```diff
@@ -1,76 +1,85 @@
 .search {
-    &_gallery {
-        margin-top: 20px;
-        display: grid;
-        grid-template-columns: repeat(2, 1fr);
-        grid-gap: 1em;
-        
-        &--image {
-            width: 100%;
-            height: 100%;
-            max-width: 100%;
-            min-height: 100%;
-            object-fit: cover;
-        }
+  &_gallery {
+    margin-top: 20px;
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    grid-gap: 1em;
 
-        &--no-image {
-            margin-top: 50px;
-            display: flex;
-            justify-content: center;
-        }
+    &--image {
+      width: 100%;
+      height: 100%;
+      max-width: 100%;
+      min-height: 100%;
+      object-fit: cover;
+    }
 
-        &--no-image-text {
-            color: $white;
-            font-size: 30px;
-        }
+    &--no-image {
+      margin-top: 50px;
+      display: flex;
+      justify-content: center;
     }
-    
-    &_gallery > div {
-        overflow: hidden;
-        position: relative;
-        height: 42vw;
-        border-radius: 8px;
-    }    
+
+    &--no-image-text {
+      color: $white;
+      font-size: 30px;
+    }
+  }
+
+  &_gallery > div {
+    overflow: hidden;
+    position: relative;
+    height: 42vw;
+    border-radius: 8px;
+  }
+}
+.expand-button {
+  display: none;
+  background-color: $darkBlue;
 }
 
 .modal-dialog {
-    .modal-body {
-        padding: 0;
-    }
+  .modal-body {
+    padding: 0;
+  }
 }
 
 .modal-image {
-    img {
-        width: 100%;
-    }
+  img {
+    width: 100%;
+  }
 }
 
-.image-wrapper:hover {
-    cursor: pointer;
+.image-wrapper:hover .expand-button {
+  position: absolute;
+  top: 1%;
+  left: 89%;
```

Comment:
```suggestion
  right: 10px;
```

Reply:
I think it would look better if this value is constant.

Reply:
you're right